lib/DDG/Fathead/MDNJS.pm

```diff
@@ -4,8 +4,8 @@ use DDG::Fathead;
 
 primary_example_queries "javascript array sort";
 secondary_example_queries
-    "js array sort",
-    "mdn array sort";
+    "js requestanimationframe",
+    "mdn localstorage";
```

Comment:
@jdorweiler it's by no means necessary but maybe we could add some of the more exotic examples the new dump has?